annotations/views.py

```diff
@@ -218,6 +218,44 @@ def list_texts(request):
     }
     return HttpResponse(template.render(context))
 
+@login_required
+def list_user(request):
+    """
+    List all the users of Vogon web
+    """
+
+    template = loader.get_template('annotations/contributors.html')
+
+    #Default Sort
+    sort = "user_name"
+    #To map the column sort parameter from UI to the column name in DB
+    sort_dict = {"user_name":"username", "name":"full_name",
```

Comment:
I would but that would be exposing the the database structure to the outside world. In this case it is not going to affect but in general its considered bad practice to do so. The mapping is just to be on the safer side of things.